### PR TITLE
Copy inputdata instead of ln -s + help section update

### PR DESCRIPTION
**Pull request by yvanlebras**

### CODE REVIEW

File: tools/interactive/interactivetool_geoexplorer.xml [hunk 1]
```diff
@@ -49,16 +49,16 @@ App: http://www.davesteps.com/geoExploreR/
 
 **Input data file MUST have as longitude column clomun 2, and as latitude column, column 3**
 
-Example input file (TAB separated)::
+Example input file (csv)::
```
Comment (bgruening) [inline tools/interactive/interactivetool_geoexplorer.xml:52]: its still tab separated, isn't it?

Comment (yvanlebras) [reply tools/interactive/interactivetool_geoexplorer.xml:52]: In fact no, it appears that it works with csv, maybe only but I need to try with tab

Comment (yvanlebras) [reply tools/interactive/interactivetool_geoexplorer.xml:52]: Or you mean I have to modify the "aspect" of the example in the help section ?

**Comment by bgruening** [comment]
ok, I trust you :)

**Comment by yvanlebras** [comment]
No, don't! NEVEERRRR ;)

**Pull request merged**